integration_tests/support/commands.ts

```diff
@@ -7,5 +7,9 @@ Cypress.Commands.add('signIn', (options = { failOnStatusCode: false }) => {
 })
 
 Cypress.Commands.add('wiremockVerify', (requestPatternBuilder: RequestPatternBuilder, expectedCount?: number) => {
-  return cy.wrap(verify(expectedCount || 1, requestPatternBuilder)).should('be.true')
```

Comment:
I think this should explicitly check for null/undefined (using `==`), as `0` is a valid argument (although it would be preferable to use the new `wiremockVerifyNoInteractions()` method I've added).